components/director/internal/domain/eventing/service.go

```diff
@@ -62,15 +62,20 @@ func (s *service) CleanupAfterUnregisteringApplication(ctx context.Context, appI
 		return nil, errors.Wrapf(err, "while deleting labels [key=%s]", labelKey)
 	}
 
-	return model.NewApplicationEventingConfiguration(EmptyEventingURL), nil
+	return model.NewEmptyApplicationEventingConfig()
 }
 
-func (s *service) SetForApplication(ctx context.Context, runtimeID uuid.UUID, appID uuid.UUID) (*model.ApplicationEventingConfiguration, error) {
+func (s *service) SetForApplication(ctx context.Context, runtimeID uuid.UUID, app model.Application) (*model.ApplicationEventingConfiguration, error) {
 	tenantID, err := tenant.LoadFromContext(ctx)
 	if err != nil {
 		return nil, errors.Wrapf(err, "while loading tenant from context")
 	}
 
+	appID, err := uuid.Parse(app.ID)
+	if err != nil {
+		return nil, errors.Wrapf(err, "while parsing application ID: %s", app.ID)
```

Comment:
Do we need it? Our DB will return error, right? I don't think we need this kind of manual check now; it's the matter of wrapping DB error

Reply:
Yes, because in internal methods we use UUID type.